docs/root/configuration/http/http_filters/ext_authz_filter.rst

```diff
@@ -147,12 +144,13 @@ The HTTP filter outputs statistics in the ``cluster.<route target cluster>.ext_a
   :header: Name, Type, Description
   :widths: 1, 1, 2
 
-  ok, Counter, Total responses from the filter.
+  ok, Counter, Total responses from the authorization service that allowed the request.
   error, Counter, Total errors contacting the external service.
-  denied, Counter, Total responses from the authorizations service that were to deny the traffic.
-  disabled, Counter, Total requests that are allowed without calling external services due to the filter is disabled.
-  failure_mode_allowed, Counter, "Total requests that were error(s) but were allowed through because
-  of failure_mode_allow set to true."
+  denied, Counter, Total responses from the authorization service that denied the request.
+  disabled, Counter, Total requests that were allowed without calling the external service because the filter is disabled.
+  failure_mode_allowed, Counter, Total error responses that were allowed through because :ref:`failure_mode_allow
+  <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.failure_mode_allow>` is set to ``true``.
```

Comment:
```suggestion
  failure_mode_allowed, Counter, "Total error responses that were allowed through because :ref:`failure_mode_allow
  <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.failure_mode_allow>` is set to ``true``."
```